modules/baseline-router-performance.adoc

```diff
@@ -1,12 +1,9 @@
 [id="baseline-router-performance_{context}"]
-= Baseline router performance
+= Baseline Ingress Controller (router) performance
 
-The {product-title} router is the Ingress point for all external traffic
-destined for {product-title} services.
+The {product-title} Ingress Controller, or router, is the Ingress point for all external traffic destined for {product-title} services.
 
-When evaluating a single HAProxy router performance in terms of
-HTTP requests handled per second, the performance varies depending
-on many factors. In particular:
+When evaluating a single HAProxy router performance in terms of HTTP requests handled per second, the performance varies depending on many factors. In particular:
 
 * HTTP keep-alive/close mode
```

Comment:
Not in scope for this PR, but thinking it would be nice to first-cap in the list, per ISG (p. 20): "Capitalize the first word of each item in a vertical list."

Reply:
Good idea and easy fix!